.env.example

```diff
@@ -32,7 +32,7 @@ GRAPHQL_URL=https://graphql-dev.dosomething.org
 # Search Engine
 ALGOLIA_APP_ID=
 ALGOLIA_SECRET=
-SCOUT_DRIVER=algolia
+SCOUT_DRIVER=
```

Comment:
Removed pre-setting this to "algolia" since Circle CI uses the .env.example file for setting its initial env variables and by having it set to "algolia" it was trying to connect and failing. Keeping it empty allows this env variable to default to "null" and eliminate the errors we were seeing with Circle CI trying to connect and send data to Algolia.